README.md

```diff
@@ -56,7 +56,11 @@ var params = { shortUrl: 'http://goo.gl/xKbRu3' };
 
 // get the long url of a shortened url
 urlshortener.url.get(params, function (err, response) {
-  console.log('Long url is', response.longUrl);
+  if (err) {
+    console.log('Encountered error', err);
```

Comment:
Confirmed on Node.js 0.10, 0.12 - space is added automatically for each argument to `console.log`.